spec/features/restrooms_spec.rb

```diff
@@ -69,23 +70,18 @@
       expect(find('button.current-location-button')['aria-label']).to be_truthy
     end
 
-    # rubocop:disable RSpec/NoExpectationExample
     it 'displays a map' do
       create(:oakland_restroom)
 
       visit root_path
       mock_location "Oakland"
       find('.current-location-button').click
-      # TODO: Figure out why this isn't working.
-      # print page.html
-      page.has_css?(".mapToggle", visible: true)
-      # find('.mapToggle').click
-      # print page.html
-
-      # TODO: Figure Out why This isn't working either
-      # page.has_css?(#)
-      # expect(page).to have_css('#mapArea.loaded')
-      # expect(page).to have_css('#mapArea .numberCircleText')
+      expect(page).to have_css(".map-toggle-btn", visible: :visible)
```

Comment:
Just documenting something here in case anyone is confused like I was.

I was a little unclear about the difference between `true` and `:visible` here, but it turns out they are equivalent, per the docs:

https://rubydoc.info/github/teamcapybara/capybara/Capybara/Node/Finders#all-instance_method

> visible (Boolean, Symbol) — Only find elements with the specified visibility. Defaults to behavior indicated by [ignore_hidden_elements](https://rubydoc.info/github/teamcapybara/capybara/Capybara#configure-class_method).
> [ . . . ]
> - true - only finds visible elements.

> [ . . . ]
> - :visible - same as true; only finds visible elements.

Reply:
Not sure why the CSS class for this button changed since the commented out test was written as `.mapToggle`, but indeed it did change to `.map-toggle-btn`! Thank you for fixing this.